test/jest/jest.config.js

```diff
@@ -7,16 +7,20 @@ module.exports = {
     ],
     "testPathIgnorePatterns": [
     ],
+    "moduleNameMapper": {
```

Comment:
this is the img  resource stub @ilmotta

Reply:
I think this fixes component tests in current `develop` branch, can we merge it first on a separate PR if this one takes too long to get merged? 

![Uploading el-gato-con-botas-1551289404.jpeg…]()

Reply:
@briansztamfater, do you how did we end up in the situation where `develop` has failing unit tests? I thought it wasn't possible. If the tests are deterministically failing, then the CI should've blocked the merge.

Any idea?

Reply:
> [...] can we merge it first on a separate PR if this one takes too long to get merged?

In my experience, when a development branch has failing tests, the collaborator who broke the tests (and who probably has more context about how to fix them) go ahead and fix them ASAP, or the persons with most knowledge of the failure. And this should be done in a separate PR to unblock other collaborators and not affect PRs with business as usual work.

So I agree with you about trying to fix `develop` as quickly as possible, but I think such fixes should always be done on separate PRs, particularly to avoid merging things in a hurry.

Reply:
For some reason many component tests are failing locally for me (by running `make component-test`), also @cammellos was experiencing this I think. I cannot say why in CI environment is working or maybe jest tests are not being checked?

Reply:
> > [...] can we merge it first on a separate PR if this one takes too long to get merged?
> 
> In my experience, when a development branch has failing tests, the collaborator who broke the tests (and who probably has more context about how to fix them) go ahead and fix them ASAP, or the persons with most knowledge of the failure. And this should be done in a separate PR to unblock other collaborators and not affect PRs with business as usual work.
> 
> So I agree with you about trying to fix `develop` as quickly as possible, but I think such fixes should always be done on separate PRs, particularly to avoid merging things in a hurry.

Yeah agree, I was about to prepare a PR with this fix but just saw that we already had fixes in this one which probably are better tested

Reply:
Yeah @briansztamfater @ilmotta component tests are not being run in ci yet.
@siddarthkay is working on this afaik.

The failing tests are a separate issue.

As to why component tests are failing - 🤷‍♂️ but maybe make sure you only have one active terminal, e.g make run-metro os not running in another.

Feel free to ping me and send the error on and I can take a look

Reply:
I believe it's caused by PR here https://github.com/status-im/status-mobile/pull/14735#issuecomment-1376799083
interface of `:toasts/upsert` changed in this pr and not all the usage of it updated
I'll fix it asap

Reply:
Oops, it's not caused by this, the usage is already updated